src/model/AvailabilityManagerAssignmentList.hpp

```diff
@@ -60,6 +60,8 @@ class MODEL_API AvailabilityManagerAssignmentList : public ModelObject {
    */
   explicit AvailabilityManagerAssignmentList(const Loop& loop);
 
+  explicit AvailabilityManagerAssignmentList(const Model& model);
```

Comment:
Can we make it protected/private? Or at least ignore in the bindings... I don't like the fact that this becomes public, as users shouldn't have any business creating orphaned ones.

Reply:
yeah. I can do that. note that even if you use `AvailabilityManagerAssignmentList(const Loop& loop)` the new AVM is still orphaned. The loop argument is actually only used to set the name of the AVM. This itself should probably be considered a bug. https://github.com/NREL/OpenStudio/blob/fa9a77bf173556a2a044f64789a7f4f01b0df076/src/model/AvailabilityManagerAssignmentList.cpp#L409

Reply:
I suppose to some extent and to echo #4072 AvailabilityManagerAssignmentList doesn't _really_ have any business having a name in the OSM since the user isn't meant to interact with it... It could be created at FT time.

Reply:
@jmarrec you are certainly welcome to add to the list of objects in #4077 to "un-name" If it is not user facing then I think it is fair game. For those that have non uuid names, and perhaps even those that do, I would anticipate a performance advantage.

Reply:
Certainly users will appreciate having nice names in the finished EnergyPlus idf. AVMlist named after the loop it belongs to etc.